src/main/webapp/readfile.html

```diff
@@ -52,7 +62,7 @@ <h2>Read Configuration</h2>
             "settlement_config": {<br>
               "matching_mode": "STRING"<br>
             },<br>
-            "product_account_key": "STRING",<br>
+            "account_id": "STRING",<br>
```

Comment:
I think this should stay "product_account_key", and "next_gen_customer_id" should be "next_gen_account_id" (9 lines up from here).